oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexMBeanImpl.java

```diff
@@ -269,6 +280,10 @@ public boolean isFailing() {
         return getFieldTermPrefixInfo(indexPath, field, Integer.MAX_VALUE, term);
     }
 
+    private void bindIndexInfoProviders(IndexInfoServiceImpl indexInfoService) {
```

Comment:
this method is called only in the constructor. To improve the readability, I would move this logic in the constructor so the reader does not have to jump somewhere else to see what it does.

Reply:
done